qiita_pet/webserver.py

```diff
@@ -93,9 +95,16 @@ def __init__(self):
             (r"/stats/", StatsHandler),
             (r"/download/(.*)", DownloadHandler),
             (r"/vamps/(.*)", VAMPSHandler),
-            # 404 PAGE MUST BE LAST IN THIS LIST!
-            (r".*", NoPageHandler)
         ]
+        if qiita_config.portal == "QIITA":
+            # Add portals editing pages only on main portal
+            portals = [
+                (r"/admin/portals/studies/", StudyPortalHandler)
```

Comment:
Yes, correct. I'd like to have it ready for extension now, instead of adding it later.